mubench.reviewsite/src/CSVHelper.php

```diff
@@ -0,0 +1,101 @@
+<?php
+namespace MuBench\ReviewSite;
+
+
+class CSVHelper
+{
+
+    function __construct()
+    {
+
+    }
+
+    public function getCSVHeaderForStatsExp($experiment)
```

Comment:
Exp = Experiment? Exp = Export? Write it out. No need for abbreviations. We have autocomplete.

Also, I'm not sure why this particular piece became a separate function?